config/locales/en.yml

```diff
@@ -430,19 +437,20 @@ en:
       search_on_this_page: "Search on this page"
       no_results: "No results"
       download: "Download:"
+
       
-  # activerecord:
-    # attributes:
-      # client:
-        # ssn: "Social security number"
-        # dob: "Date of birth"
-        # num_in_household: "Number in household"
-        # num_of_dependants: "Number of dependants"
-    # errors:
-    #   models:
-    #     client:
-    #       attributes:
-    #         ssn:
-    #         dob:
-    #         num_of_dependants:
-    #         num_in_household:
```

Comment:
Hmm, interesting. I understand we you're removing this, it's not being used, right? But if we wanted to add the internationalization to the active record error messages, is this How'd we do it?  Did we ever try it?